### PR TITLE
Add self deducted proish variant-

### PR DESCRIPTION
**Pull request by gobijan**
In case you decide that it's okay for you to have a self deducted proish color variant: here is a pull request of my take on it.
In the end it's just a combination of colors.

### CODE REVIEW

**Comment by facelessuser** [comment]
I do actually already have all pro-ish versions completed in a private repo. Along with support in Merge:

Text - Buffy
Text - Pro
Text - Van Helsing

Your interpretation is very close to mine though give or take a little here and there. I think you came up with a slightly different red than I.

Backgrounds are probably ever so slightly different, but close enough. Comments, line highlights etc. also are close but slightly different.

Overall, the differences are probably not noticeable to the eye too much.

**Comment by facelessuser** [comment]
I didn't post all screenshots, but just enough to illustrate the point 🙂.

**Comment by facelessuser** [comment]
If it helps, I make the selection a variant of the background (line selection as well). Play with saturation and lightness.

Comments usually are a reflection of the overall color of the variant, from what I can tell. So Pro gives a very purplish feel, so I make comments an adjusted purple. Van Helsing, though very dark, seems to use cyan as the main accent, so comments reflect that. Blade, though at first blush, it may seem the theme is greenish, from what I can tell, comments are also a variant of cyan. Across all variants, I pretty much use the same saturation and lightness adjustments for comments, just swap out the target color.  The same goes for line highlights, etc. (though, obviously, I use a different adjustment combo than I do for comments).

**Comment by facelessuser** [comment]
Text - Lincoln
Text - Morbius
Text - Blade

I also did a slight variation on the classic that follows similar handling of comments selections and such as the pro vairants. I actually prefer this approach to selections and line highlights for classic over what the original open source one defines. Though, I do believe I keep the classic comment color unaltered.

Text - Classic

**Comment by facelessuser** [comment]
In general, I purposely made the color scheme easy to modify and make a pro-ish variant. I even give enough info in the readme to adjust the default to a pro-ish without giving away what I think the exact saturation, lightness adjustments are. I also don't give the exact backgrounds or core colors. So far, this is my compromise.

At some point, I would love to pull in the Pro work I have sitting in my private repo, but for now, I'm going to give it time before and see what others do before I make that jump.

Basically, I'd like to see if people are safely releasing the color palette. I'd never release the actual work source that is being sold, but if people are safely getting out and sharing the palette, then I can release my work here which is not based on any sold source, just the color palette.

**Comment by gobijan** [comment]
I see and I can perfectly understand you :) Thx for your work on the adoption based on Mariana :)
